lib/mocha/any_instance_method.rb

```diff
@@ -28,5 +20,9 @@ def stub_method_body(method_name)
     def original_method_owner
       stubbee
     end
+
+    def instance
```

Comment:
I'm not sure about the method name, `#instance`, either. Maybe something like `#mock_owner` would be better...? What do you think?